VoodooI2C/VoodooI2C/VoodooI2CDevice/VoodooI2CDeviceNub.cpp

```diff
@@ -114,6 +114,7 @@ IOReturn VoodooI2CDeviceNub::getDeviceResourcesDSM(UInt32 index, OSObject **resu
 
     UInt8 availableIndex = *(reinterpret_cast<UInt8 const*>(data->getBytesNoCopy()));
     data->release();
+    *result = nullptr;
 
     if (!(availableIndex & (index << 1))) {
```

Comment:
While somewhat unrelated to this PR, shouldn't this be `1 << index`?

Reply:
Many thanks, almost overlooked that!